src/app/modules/button/button.component.css

```diff
@@ -0,0 +1,3 @@
+button {
+  max-width: 300px;
```

Comment:
I'm not a huge fan of limiting the width of all buttons.  If this is just for buttons inside of tables, could you make this change in the way the table component includes buttons?